### PR TITLE
Proposed new version

### PR DESCRIPTION
**Pull request by cconcolato**
This proposed new version is the result of major restructuring and simplification of the spec text, with some technical clarifications.

### CODE REVIEW

**Comment by cconcolato** [comment]
Browsable version here https://rawgit.com/AOMediaCodec/av1-avif/major_restucturing/docs/

File: index.bs [hunk 1]
```diff
@@ -36,564 +36,122 @@ THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
...
-The image data should have the "reduced_still_picture_header" field set to 1 in the sequence header OBU as specified in section 5.5.1 of the Bitstream & Decoding Process Specification [[!AV1]]. The field may be set to 0 to allow for easy extraction of frames from an AV1 file.
-</p>
-
-<h2 id="avif=image-collection">AVIF Image Collection</h2>
-
-<p>The image data of type "av1i" shall be used for an image collection item coded with AV1.
-
-The image item data shall be structured as defined in the AV1 Sample Format section of the AV1 Codec ISO Media File Format Binding
-[[!AV1]] specification.
+<p>When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>, it is called an <dfn>AV1 Image Item</dfn>, and SHALL obey the following constraints:
```
Comment (lambdapioneer) [inline index.bs]: Can you share the motivation behind the naming to `av01` compared to `av1i`?

File: index.bs [hunk 2]
```diff
@@ -36,564 +36,122 @@ THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
...
-</p>
-
-<h3 id="ver1-derived-images">Derived Images</h3>
-
-<p>The derivation of tiled images may be supported by Baseline readers which also support rendering images
-larger than permited by AV1 profiles. No other derived image format need be supported.
-</p>
+<p>An <dfn>AV1 Image Sequence</dfn> is defined as a set of AV1 [=Temporal Units=] stored in an [=AV1 track=] as defined in [[!AV1-ISOBMFF]] with the following constraints:
+<ul>
+<li>The [=Sequence Header OBUs=] SHALL have their <code>[=still_picture=]</code> flag set to 1 and SHOULD have their <code>[=reduced_still_picture_header=]</code> flags set to 1.
```
[the remark this answers is not in the document]
Comment (cconcolato) [reply index.bs]: I don't see why we would need timing information inband when the ISOBMFF layer has it. 

Anyway, this text will probably change based on feedback from #12

File: index.bs (hunk 1 above)
Comment (cconcolato) [reply index.bs]: Definitely a discussion we could have. See also #9.

Annex J of HEIF says:
> It is suggested that image sequence tracks are specified like video tracks of the same coding format with a change of handler type (from 'vide' to 'pict') and possibly with additional constraints.

and
>In coded image items, the coding type is identified by the item_type, and in image sequences, by the sample entry type. It is suggested that the same code be used for the two cases;

I just followed these suggestions.

**Pull request merged**